Sources/SignalRClient/HubConnection.swift

```diff
@@ -26,6 +26,9 @@ public class HubConnection {
     private var connectionDelegate: HubConnectionConnectionDelegate?
     private var hubProtocol: HubProtocol
 
+    public var keepAliveIntervalInSeconds: Double = 15
+    private var keepAlivePingTask: DispatchWorkItem?
```

Comment:
We don't seem to ever clean this up and we are pretty much guaranteeing this will fire even after the connection is stopped - either gracefully or because of an error.

Reply:
I added some cleanup and a test to control.